packages/core/src/queries/logto-config.ts

```diff
@@ -39,13 +44,18 @@ export const createLogtoConfigQueries = (pool: CommonQueryMethods) => {
         where ${fields.key} in (${sql.join(keys, sql`,`)})
     `);
 
-  const updateOidcConfigsByKey = async (key: LogtoOidcConfigKey, value: OidcConfigKey[]) =>
-    pool.query(sql`
+  const updateOidcConfigsByKey = async (key: LogtoOidcConfigKey, value: OidcConfigKey[]) => {
+    // Set a expiration timestamp in redis cache, and check it before returning the tenant LRU cache. This helps
+    // determine when to invalidate the cached tenant and force a in-place rolling reload of the OIDC provider.
+    void wellKnownCache?.set('tenant-cache-expires-at', WellKnownCache.defaultKey, Date.now());
```

Comment:
will it be better to build a function like `.invalidate()` for `TenantContext`?

Reply:
because unlike other cached queries, this action is not related to the cache itself, i.e. not able to use `.memoize`.

Reply:
Updated!!!!